ext/wry_ruby/src/application/tray_id.rs

```diff
@@ -0,0 +1,50 @@
+use magnus::{function, Error, ExceptionClass, Module, Object, RString};
+use std::cell::RefCell;
+use wry::application::TrayId as TrayIdImpl;
+
+#[derive(Clone, Debug)]
+#[magnus::wrap(class = "TrayId")]
+pub struct TrayId {
+    inner: Option<RefCell<TrayIdImpl>>,
```

Comment:
I'm not sure I understand why this is a `RefCell` or even an`Option`. If the string is empty we could either return an error or use a default.

Reply:
i remember using  TrayIdImpl , which lead me into errors like  unsatisfied traits typedData and all. n then when i was trying to check empty or not also it didn't let me so i ended dng it so i get better idea on what's going on n  mutability.  

will update it!